test/int-test-common.sh

```diff
@@ -155,7 +155,7 @@ backup_regdomain() {
 set_regdomain() {
 	if [ -z "$1" ]; then
 		echo "set_regdomain(): No regdomain given"
-		retun 1
+		return 1
```

Comment:
I'm quite puzzled to see that it has been there for quite a while before being noticed. Is this something never included anywhere?